packages/modal/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "pcln-modal",
-  "version": "2.0.2",
+  "version": "2.0.3",
```

Comment:
@unscsprt,

Would you mind reverting the version change in package.json of modal package? The current publish workflow stamps version in master by Collaborator. (Thought behind it is that there could be other concurrent PRs that can be published in the to-be version)